pegasus/emails/hoc_signup_2017_receipt_en.md

```diff
@@ -1,39 +1,47 @@
 ---
 from: "Hadi Partovi (Code.org) <hadi_partovi@code.org>"
-subject: "Thanks for signing up to host an Hour of Code!"
+subject: "Get ready for the Hour of Code"
 litmus_tracking_id: "5g5lyi1a"
 ---
-<% hostname = CDO.canonical_hostname('hourofcode.com') %>
+  <% hourofcode = CDO.canonical_hostname('hourofcode.com') %>
+  <% codedotorg = CDO.canonical_hostname('code.org') %>
+  <% storedotcodedotorg = CDO.canonical_hostname('store.code.org') %>
 
-# Thanks for signing up to host an Hour of Code!
+### Thanks for signing up to host an Hour of Code!
+As a thank you for helping make it possible for students to start learning computer science, we'd like to give you a [free set of 12 professionally printed posters featuring diverse role models for your classroom](https://<%= storedotcodedotorg %>/products/code-org-posters-set-of-12). Use offer code FREEPOSTERS at checkout. (This code is good through 12/31 while supplies last. You'll need to cover shipping costs. If that is not in your budget, the [PDF files](https://<%= codedotorg %>/educate/resources/inspire#posters) are available to print yourself.)
```

Comment:
We should link the PDF files to `https://hourofcode.com/promote/resources#posters` because these include the localized versions.

Reply:
Changed.